chapter_natural-language-processing-applications/natural-language-inference-bert.md

```diff
@@ -372,6 +370,8 @@ d2l.train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs, devices,
 lr, num_epochs = 1e-4, 5
 trainer = torch.optim.Adam(net.parameters(), lr=lr)
 loss = nn.CrossEntropyLoss(reduction='none')
+# A dry run to initialize the model.
```

Comment:
For now we do not include . at the end of a code comment. To save space, consider using inline comments with 2 space between the end of code and the beginning of inline comments.

Reply:
it is resolved. Thanks!